services/ui-src/src/components/pages/ReviewSubmit/AdminReview.test.tsx

```diff
@@ -81,6 +81,17 @@ describe("MFP WP Review and Submit Page Functionality", () => {
       const { adminInfo } = review;
       expect(screen.getByText(adminInfo.submitLink.text)).toBeEnabled();
     });
+
+    test("should not show console errors", () => {
```

Comment:
This kind of test might not work on all areas right now, because console errors around the app are buried in different components sometimes, but because these errors are specific to the Review page, I wanted to add these as something to aspire to.